views/pages/layouts/examples/gallery.handlebars

```diff
@@ -1,7 +1,7 @@
 {{setTitle    "Photo Gallery"}}
 {{setSubTitle "A layout example that shows off a responsive photo gallery."}}
 {{setTags     "base" "grids" "forms" "buttons" "menus"}}
-{{addLocalCSS "/css/layouts/gallery-grid.css" hasOldIE=true}}
+{{addRemoteCSS "https://yui-s.yahooapis.com/pure/0.6.0/grids-responsive-min.css"}}
```

Comment:
For the 0.6.0 part, can you use the `{{pure.version}}` variable so the version gets updated as pure releases new versions?